### DISCUSSION TITLE
Use Foundry better [3]

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2552] [Ideas]
GM @PatrickAlphaC 
---
![image](https://github.com/user-attachments/assets/5948ce83-af5f-4d2d-b960-b5755dd24837)
---
At Advanced Foundry Course, Section2, EP 7, you told use that we can compare two strings using this method, but i wanted to say there is an another way to compare everything ( equality ) using `assertEq`
---
![image](https://github.com/user-attachments/assets/ea90d2a2-3052-4635-b8b4-aea9dc13479c)
<br>
![image](https://github.com/user-attachments/assets/1c804653-0335-4133-96eb-ccd5210b147e)
---

**Comment by EngrPips** [comment]
Interesting. Thanks for the heads-up buddy.

**Reply by PatrickAlphaC** [reply]
Nice! This is new. We should add this.

**Reply by EngrPips** [reply]
Yeah wasn't a thing at the course creation time.

**Reply by EngrPips** [reply]
To me it feels like an abstraction of the `keccack` stuff, but maybe I am wrong lol

**Reply by MostPow3rful** [reply]
You are right Sir
---
![image](https://github.com/user-attachments/assets/98c62bda-4bbc-4efd-984c-6e6f46aa48d8)

**Reply by EngrPips** [reply]
Interesting
